### ISSUE TITLE
NPE in AuthorizationFragment.createCustomTabResponseIntent

### ISSUE THREAD

**Issue by mellanox-is** [AzureAD/microsoft-authentication-library-for-android#943]
MSAL 1.2.0 exception:

```
java.lang.RuntimeException: 
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3430)
  at android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:3614)
  at android.app.servertransaction.LaunchActivityItem.execute (LaunchActivityItem.java:86)
  at android.app.servertransaction.TransactionExecutor.executeCallbacks (TransactionExecutor.java:108)
  at android.app.servertransaction.TransactionExecutor.execute (TransactionExecutor.java:68)
  at android.app.ActivityThread$H.handleMessage (ActivityThread.java:2199)
  at android.os.Handler.dispatchMessage (Handler.java:112)
  at android.os.Looper.loop (Looper.java:216)
  at android.app.ActivityThread.main (ActivityThread.java:7625)
  at java.lang.reflect.Method.invoke (Native Method)
  at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run (RuntimeInit.java:524)
  at com.android.internal.os.ZygoteInit.main (ZygoteInit.java:987)
Caused by: java.lang.NullPointerException: 
  at android.content.ComponentName.<init> (ComponentName.java:131)
  at android.content.Intent.<init> (Intent.java:6216)
  at com.microsoft.identity.common.internal.providers.oauth2.AuthorizationFragment.createCustomTabResponseIntent (AuthorizationFragment.java:185)
  at com.microsoft.identity.client.BrowserTabActivity.onCreate (BrowserTabActivity.java:67)
  at android.app.Activity.performCreate (Activity.java:7458)
  at android.app.Activity.performCreate (Activity.java:7448)
  at android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1286)
  at android.app.ActivityThread.performLaunchActivity (ActivityThread.java:3409)
```

**Comment by shahzaibj** [comment]
@mellanox-is Do you have repro steps for this issue? Any MSAL logs you can provide us to help investigate?

**Comment by shahzaibj** [comment]
@mellanox-is I am not able to reproduce the issue, however, I feel that this issue may be related to some of the issues fixed by this PR: https://github.com/AzureAD/microsoft-authentication-library-common-for-android/pull/828

We are currently going through our test pass and will be releasing the next version of MSAL with those bits later this week. I will provide an update once the release is in Maven.

**Comment by mellanox-is** [comment]
Thank you

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by iambmelt** [comment]
Please use an informative name when filing issues and include repro steps as well as device, environment, and other relevant details

**Comment by shahzaibj** [comment]
@mellanox-is Can you test on the latest MSAL version `1.4.0` and see if you still have the issue?:

**Comment by mellanox-is** [comment]
I will, thanks

Get Outlook for Android<https://aka.ms/ghei36>

________________________________

**Comment by shahzaibj** [comment]
Closing this as it is expected to be fixed by MSAL 1.4.0 - @mellanox-is Feel free to reopen if you are still experiencing issues.
